<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTxtDialog</class>
 <widget class="QDialog" name="QTxtDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>419</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>419</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QTxtDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#QTxtDialog
{
	border-image: url(:/SeedUI/Resources/SeedUI/PC_Camera.png);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget
{
	background-color: rgb(34, 36, 40);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#widget_2
{
	background-color: rgb(43, 43, 43);
}
</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>8</y>
           <width>54</width>
           <height>12</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(234, 126, 24);
background-color: rgb(43, 43, 43);</string>
         </property>
         <property name="text">
          <string>字幕编辑</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>8</y>
           <width>15</width>
           <height>14</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/SeedUI/Resources/SeedUI/SeedUI_txt.png);
background-color: rgb(43, 43, 43);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Close_pushButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>395</x>
           <y>10</y>
           <width>10</width>
           <height>10</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#Close_pushButton{
	background-color: rgb(43, 43, 43);
	border-image: url(:/Close/Resources/Close_Normal.png);
}
QPushButton#Close_pushButton:hover{
	background-color: rgb(43, 43, 43);
	border-image: url(:/Close/Resources/Close_Hover.png);
}
QPushButton#Close_pushButton:disabled{
	background-color: rgb(43, 43, 43);
	border-image: url(:/Close/Resources/Close_Disabled.png);
}
QPushButton#Close_pushButton:checked{
	background-color: rgb(43, 43, 43);
	border-image: url(:/Close/Resources/Close_checked.png);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFontComboBox" name="fontComboBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox#fontComboBox {
    background-color: black;
    border: 1px solid rgb(100, 100, 100);
    color: white;
    padding:1px 2em 1px 3px;
	font: 75 14px &quot;微软雅黑&quot;;
	border-radius:5px;
	color: rgb(254, 161, 24);
}
QComboBox#fontComboBox:hover {
    background-color: rgb(24, 24, 24);
    border: 1px solid #333333;
    color: white;
    padding:1px 2em 1px 3px;
	font: 75 14px &quot;微软雅黑&quot;;
	border-radius:5px;
	color: rgb(254, 161, 24);
}
QComboBox#fontComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    border-left: 1px solid #333333;
	width: 25px;
}/* 下拉按钮 */
QComboBox#fontComboBox::drop-down:hover {
  	subcontrol-origin: padding;
    subcontrol-position: top right;
    border-left: 1px solid #333333;
	background-color: rgb(83, 83, 83);
}/* 下拉按钮 鼠标移动 */
QComboBox#fontComboBox::down-arrow {
    border: 1px solid #333333;
    width: 8px;
    height: 5px;
    background: rgb(83, 83, 83);
	border-bottom-left-radius: 5px; 
	border-bottom-right-radius: 5px; 
}/* 下拉箭头 */
QComboBox#fontComboBox::down-arrow:hover {
    border: 1px solid #333333;
    width: 8px;
    height: 5px;
    background: rgb(170, 170, 170);
	border-bottom-left-radius: 5px; 
	border-bottom-right-radius: 5px; 
}/* 下拉箭头 鼠标移动 */
QComboBox#fontComboBox QAbstractItemView{
	outline:0px;/*去掉虚线*/
	color: rgb(209, 209, 209);
	font: 75 14px &quot;微软雅黑&quot;;
	background-color: rgb(43, 43, 43);
	 selection-background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}/* 下拉选择 */
QComboBox#fontComboBox QAbstractItemView::item{height:20px;}/*下拉选择高度*/</string>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="butTexItalic">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#butTexItalic
{
	border-image: url(:/SeedUI/Resources/SeedUI/Italic.png);
}

QPushButton#butTexItalic:hover
{
	border-image: url(:/SeedUI/Resources/SeedUI/Italic_H.png);
}

QPushButton#butTexItalic:checked
{
	border-image: url(:/SeedUI/Resources/SeedUI/Italic_C.png);
}

QPushButton#butTexItalic:checked:hover
{
	border-image: url(:/SeedUI/Resources/SeedUI/Italic_C_H.png);
}

QPushButton#butTexItalic:disable
{
	border-image: url(:/SeedUI/Resources/SeedUI/Italic_D.png);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="butTexBold">
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton#butTexBold
{
	border-image: url(:/SeedUI/Resources/SeedUI/Overstriking.png);
}

QPushButton#butTexBold:hover
{
	border-image: url(:/SeedUI/Resources/SeedUI/Overstiking_H.png);
}

QPushButton#butTexBold:checked
{
	border-image: url(:/SeedUI/Resources/SeedUI/Overstriking_C.png);
}

QPushButton#butTexBold:checked:hover
{
	border-image: url(:/SeedUI/Resources/SeedUI/Overstriking_C_H.png);
}

QPushButton#butTexBold:disable
{
	border-image: url(:/SeedUI/Resources/SeedUI/Overstriking_D.png);
}

</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="butTexSizeByWidth">
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#butTexSizeByWidth
{
	border-image: url(:/SeedUI/Resources/SeedUI/RollingMode.png);
}

QPushButton#butTexSizeByWidth:hover
{
	border-image: url(:/SeedUI/Resources/SeedUI/RollingMode_H.png);
}

QPushButton#butTexSizeByWidth:checked
{
	border-image: url(:/SeedUI/Resources/SeedUI/RollingMode_C.png);
}

QPushButton#butTexSizeByWidth:checked:hover
{
	border-image: url(:/SeedUI/Resources/SeedUI/RollingMode_C_H.png);
}

QPushButton#butTexSizeByWidth:disable
{
	border-image: url(:/SeedUI/Resources/SeedUI/RollingMode_D.png);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="butTexColor">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="dspScrollCharPerSecond">
          <property name="styleSheet">
           <string notr="true">QDoubleSpinBox#dspScrollCharPerSecond {
        border-radius:3px;
        height: 24px;
        min-width: 40px;
        border: 1px solid rgb(100, 100, 100);
        background: rgb(20, 20, 20);
 		color: rgb(254, 161, 24);
}
QDobleSpinBox#dspScrollCharPerSecond :enabled {
       color: rgb(254, 181, 24);
}
QDobleSpinBox#dspScrollCharPerSecond :enabled:hover, QLineEdit:enabled:focus {
       color: rgb(254, 181, 24);
}
QDobleSpinBox#dspScrollCharPerSecond :!enabled {
        color: rgb(255, 255, 255);
        background: transparent;
}</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>9.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.200000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>5</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QTextEdit" name="texInput">
          <property name="styleSheet">
           <string notr="true">QTextEdit
{
  border-radius:5px;
  border: 1px solid rgb(100, 100, 100);
  background-color: rgb(33, 33, 33);
  color: rgb(254, 161, 24);
  font: 14px &quot;微软雅黑&quot;;
}</string>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:14px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>5</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>175</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#pushButton
{
	border-image: url(:/SeedUI/Resources/SeedUI/SmallButtonYellow.png);
	font: 12px &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
}

QPushButton#pushButton:hover
{
	border-image: url(:/SeedUI/Resources/SeedUI/SmallButtonYellow_H.png);
}

QPushButton#pushButton:checked
{
	border-image: url(:/SeedUI/Resources/SeedUI/SmallButtonYellow_C.png);
}
</string>
           </property>
           <property name="text">
            <string>确认</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>174</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rdzbdemo.qrc"/>
 </resources>
 <connections/>
</ui>
